content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Running Open 3D Multiplayer Projects
+description: How to run Multiplayer enabled projects and relevant configuration
+linkTitle: Running Multiplayer
+---
+
+Running projects using the Open 3D Engine Multiplayer Gem requires build steps followed by a variety of options to run the projects.
+
+* Ensure your project is [properly configured](./configuration.md)
+* Build your project's `GameLauncher` and `ServerLauncher`
+* Use one of the following methods to run
+
+## In Editor Testing
+
+Multiplayer projects can be [run in Editor](./test-in-editor.md) using Ctrl-G. See the accompanying documentation for more detail. 
+
+## Running locally using ImGui Options
+
+1. Launch your `ServerLauncher` either from Visual Studios or from your build directory.
+2. Press the `HOME` key to enable the debug menu.
+3. Select the `Host` option.
+4. Select a level to load to begin hosting.
+4. Use the `Launch Local Client` option to automatically launch a client connected to the server.
+
+## Running Server and Client Launchers and configuring via console 
+
+You can manually launch the executables for Client and Server and configure them using the application's console command line.
+
+1. Launch both `ClientLauncher` and `ServerLauncher`
+2. In `ServerLauncher` press `~` to open the command line.
+3. Run the command `host` to begin hosting.
+4. Run the command `LoadLevel <path to level>` to load a level.
+5. In `ClientLauncher` press `~` to pen the command line.
+6. Run the command `connect <IP Address>` to connect to the server. If running locally, `connect` will default to localhost.
+
+## Running Server and Client Launchers and configuring via cvar file
+
+You can manually launch the executables for Client and Server and pass a pre-defined configuration file to each for them to execute on launch. Commands in these cfgs files are executed in listed order.
```

Comment:
+1 for calling out "executed in listed order". I remember that being questioned during workflow tests :)